### ISSUE TITLE
Something wrong with TressFX and HairWorks functionality

### ISSUE THREAD

**Issue by Susie1818** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#659]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Game is not cracked, modded and use the latest patch

### Game [Required]

Tomb Raider (2013)

### Game Platform [Required]

- [X] Steam
- [ ] Microsoft Store
- [ ] GOG Galaxy
- [ ] Ubisoft Connect
- [ ] Battle.net
- [X] Epic Games
- [ ] Rockstar Games Launcher
- [ ] EA app
- [ ] Amazon Games app
- [ ] Other (please specify)

### Processor / Processor Number [Required]

intel Core i5-13500

### Graphic Card [Required]

intel Arc A770

### GPU Driver Version [Required]

31.0.101.5081

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [X] DirectX12
- [X] DirectX11
- [ ] DirectX10
- [ ] DirectX9

### Windows Build Number [Required]

- [ ] Windows 11 23H2
- [ ] Windows 11 22H2
- [ ] Windows 11 21H2
- [X] Windows 10 22H2
- [ ] Windows 10 21H2
- [ ] Other (Please specify)

### Intel System Support Utility report

[SSU_20240102.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/13809381/SSU_20240102.txt)

### Description and steps to reproduce [Required]

TressFX and HairWorks are both extensions for rendering hair and fur.

In «Tomb Raider (2013)», the TressFX feature used to work on my Arc GPU without any problem, but I just found out that it now causes severe framerate hit with the recent driver. I cannot be sure which driver version this issue started to occur with because I haven't played this game for quite a while, but I am simply very sure that there wasn't such an issue several months ago.

Using driver version 5081, the in-game benchmark gets 320 FPS with TressFX OFF and drops to only 68 FPS with TressFX ON. I am quite sure it didn't behave like this with certain older drivers. I used to play this game on my Arc GPU with TressFX ON at higher than 120 FPS stable.

======
Another related problem occurs with «Metro Exodus Enhanced Edition». If the "HairWorks" option in the graphics settings is turned ON, then the game would randomly crash very quickly. This issue has been verified and confirmed on the IntelArc subreddit community.

======
In summary, please examine and fix both TressFX and HairWorks implementations in the Arc driver. I observed the problems with these two games, but I guess the issue may also affect other games that use these two hair rendering techniques.

### Game graphic quality [Required]

- [ ] Low
- [ ] Medium
- [ ] High
- [X] Ultra

### Game resolution [Required]

1920x1080

### Game VSync [Required]

Off

### Game display mode [Required]

- [X] Fullscreen
- [ ] Windowed
- [ ] Borderless Windowed

### Detailed game settings [Required]

* Tomb Raider (2013)
"Ultimate" preset = TressFX ON
"Ultra" preset = TressFX OFF

* Metro Exodus Enhanced Edition
"Ultra" preset

**Comment by Felipe-Intel** [comment]
Hello @Susie1818 this issue is being investigated under bugID 14020085081. As soon as I have news, I'll come back to you.

**Comment by Susie1818** [comment]
@Felipe-Intel 

Do you really understand what you are talking about? Are you a bot or something?

The fact is: I just checked with the latest driver v5333, and the two issues have both been fixed. The performance in «Tomb Raider (2013)» with TressFX on has come back to the level I used to have with older drivers, 124FPS (in-game benchmark). The HairWorks feature in «Metro Exodus Enhanced Edition» also works normally now (I just played for an hour and no crash anymore).

So the "bugID 14020085081" as you mentioned has been resolved by the driver software engineers, but you just wrote something totally nonsense. The problem is technically fixable on the driver level and has been fixed as a matter of fact, okay? Do you have communication problem with your driver dev team?

This case CAN be closed but NOT because of the reason you wrote!
